### DISCUSSION TITLE
Warning

### DISCUSSION THREAD

**Discussion by HyakuzaO** [kyleskom/NBA-Machine-Learning-Sports-Betting#408] [Q&A]
![imagem](https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/160853868/d7a4eabb-7aac-4f09-9b23-454d0696a435)

Hello guys, what is supposed to mean that warning?  

It´s afects the accuarcy of the script or its irrelevant?  

Anyway to fix it?

Many Thanks guys

**Comment by nealmick** [comment]
what do you mean accuracy??  it should work, but does not make money or beat vegas odds by itself.

**Comment by HyakuzaO** [comment]
Maybe you didn't understand my question well, or I explained it poorly, I apologize. I just wanted to know if this error affects the accuracy of the model's results and how to fix the error.

**Reply by kyleskom** [reply]
No it doesn't affect anything.

**Comment by Scandianis** [comment]
It is just informing the developer/user of the xgboost software that in the future the method "is_sparse" will be removed and replaced by the method "is_instance" for the same effect. Has no effect on accuracy. It is not an error, just a warning.

The file requirements.txt in root folder of the project is stating version 2.0.0 of xgboost. 

Upgrading to latest version 2.0.3 of xgboost (https://pypi.org/project/xgboost/) may perhaps use method "is_instance" as the warning is about.

The command...
Python\Python311\Scripts\pip show xgboost

... shows also the version of xgboost in your system.
